woocommerce/woocommerce
# Replace the default image of the Testimonial single pattern

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR replaces the image for the `Testimonials single` pattern.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new page or post.
2. Insert the `Testimonials single` pattern.
3. Check them image it looks like 👇 
<img width="815" alt="Screenshot 2024-05-16 at 10 58 41" src="https://github.com/woocommerce/woocommerce/assets/186112/c3393ca3-ef19-436b-a853-d9de64b48024">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Replace the `Testimonials single` default image.
#### Comment 

</details>
